modules/host/negotiaterevisecontract.go

```diff
@@ -92,6 +93,9 @@ func (h *Host) managedRevisionIteration(conn net.Conn, so *storageObligation, fi
 				if uint64(len(modification.Data)) != modules.SectorSize {
 					return errBadSectorSize
 				}
+				if err := randomnessTest(modification.Data); err != nil {
+					return err
+				}
```

Comment:
I think inlining this function is clearer:
```go
if shannonEntry(modification.Data) < modules.MinimumEntropy {
    return errLowEntropy
}
```

Reply:
This is reserved for the future: if we decide to add more randomness tests or change existing ones, we will change `randomnessTest` function instead of modifying the inlined implementation in two places (there is a risk that we update only one place and forget to update another).

Reply:
For example, if we decide to add compression test, this function will be a good place to put it.

Reply:
It makes more sense to provide the compressed size in bits not a ratio.  Also the size before compression in bits. Ratio does not provide the same amount of information by itself.